CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(APPLE)
 	set_property(DIRECTORY
 		APPEND
 		PROPERTY COMPILE_DEFINITIONS SC_DARWIN)
+	set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install")
```

Comment:
It installs *by default* to a subdirectory "Install" of the *build directory*, wherever you chose that to be. This is how it used to be and the Readme still reflects that. You can chose any install location by adding the parameter -DCMAKE_INSTALL_PREFIX as cmake argument, and you can also move the SuperCollider folder wherever you want after the "install". 
Without this change cmake would "install" to /usr/local which simply is an awkward place for an application bundle, you can't even access it from the Finder. People who wanted the install in some visible, easily accessible place, had to make the "install_prefix" explicit. 
If you don't "install" but just work with the bundles in .../editors/sc-ide/Debug or Release this change doesn't affect you, and if you want to "install" to another location, you'd have to use the cmake argument anyways. So I agree with J. that it's a minor detail, just a convenience for people who expect a install folder with exactly the same content as the application installed via a disk-image and are used to the buildfolder/Install location.